tasks/main.yml

```diff
@@ -1,10 +1,12 @@
 ---
 - name: Validate Operating Systen
   assert:
-    that: ansible_os_family == 'Windows' or ansible_distribution in ['Debian', 'Ubuntu', 'RedHat', 'CentOS', 'SLES', 'openSUSE', 'SuSE', 'SLES_SAP', 'Windows']
+    that: >
+      ansible_os_family == 'Windows' or ansible_distribution in ['Debian', 'Ubuntu', 'RedHat', 'CentOS', 'Amazon', 'SLES', 'openSUSE', 'SuSE', 'SLES_SAP',
+      'Windows']
     msg: >
       Received invalid Operating System: '{{ ansible_distribution }}'. The Cloud Ops Ansible role supports the following OSs: 'Debian', 'Ubuntu', 'RedHat',
-      'CentOS', 'SLES', 'openSUSE', 'SuSE', 'SLES_SAP' and 'Windows'.
+      'CentOS', 'Amazon', 'SLES', 'openSUSE', 'SuSE', 'SLES_SAP' and 'Windows'.
```

Comment:
The Logging and Monitoring Agents support Amazon EC2 VMs. But Ops Agent does not support Amazon EC2 VMs. I guess the tests might fail for Ops Agent.

Reply:
Yup, I've added an exclude option to our test matrix so the ops-agent tests never run with AmazonLinux2.
https://github.com/GoogleCloudPlatform/google-cloud-ops-agents-ansible/pull/52/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR62

Reply:
Oops, missed the last email update. Just LGTM'd. Thanks for adding this!